namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml

```diff
@@ -0,0 +1,34 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: nart-team-portforward
+  namespace: offender-categorisation-dev
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - "pods/portforward"
+    verbs:
+      - "create"
+  - apiGroups:
+      - ""
+    resources:
+      - "pods"
+    verbs:
+      - "get"
+      - "list"
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nart-team-portforward
+  namespace: offender-categorisation-dev
+subjects:
+  - kind: Group
+    name: "github:dps-nart-team-access"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
```

Comment:
Can you change it to kind: Role please

Reply:
For other two namespaces as well please

Reply:
Good spot. Yes I'll fix.

Reply:
Now fixed on all 3 namespaces in this PR.